R/info.R

```diff
@@ -0,0 +1,21 @@
+#' Get information about PROOF server
+#'
+#' @export
+#' @references <https://github.com/FredHutch/proof-api?tab=readme-ov-file#get-info>
+#' @return A list with fields:
+#' - `logged_in_user` (character): NULL unless authentication header is sent
```

Comment:
The user can only do this with the current code by doing 

```r
library(proofr)
library(httr)
token <- proof_authenticate(...)
with_config(
  add_headers(Authorization = paste0("Bearer ", find_token(token))), 
  proof_info()
)
```

That's doable, but not consistent with the way we've thus far had users pass tokens

Reply:
Hmm....I'm not sure what we should do. This endpoint does not require authentication, and the only difference in the output if a token is provided is that the `logged_in_user` key of the returned object will contain your username instead of NULL. 
I'm not sure that is useful. This should never return anything unexpected - a user should never be using another user's token. 

Maybe I should take `logged_in_user` out of the output of the /info endpoint. 

What do people think?

Reply:
Yeah I think it makes sense to remove user information in this endpoint since it sounds like this endpoint is about getting information on the PROOF server, and nothing to do with users

Reply:
@dtenenba poke ... Still waiting on the change to the API endpoint to not return `logged_in_user`

Reply:
@sckott -- Just removed `logged_in_user` from the `dev` branch of proof-api if you want to implement this.